package.json

```diff
@@ -26,7 +26,7 @@
     "http-proxy-agent": "^7.0.2",
     "https-proxy-agent": "^7.0.4",
     "jszip": "^3.10.1",
-    "ora": "^8.0.1",
+    "ora": "^7.0.1",
```

Comment:
Downgraded this because ora 8 requires Node 18 and we're still compatible with Node 16 in this module.